smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/ejabberd/request/InformationRequest.java

```diff
@@ -0,0 +1,40 @@
+package hu.unideb.smartcampus.service.ejabberd.request;
+
+import java.io.Serializable;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+/**
+ * Information request can be used to request information about given group on given host.
+ *
+ */
+@Data
+@EqualsAndHashCode(callSuper = false)
+@NoArgsConstructor
```

Comment:
I think we should provide only one way to initialize our domain objects.
(The builder pattern is the correct way now.)
It's important because we want to create (in every layer except persistence) immutable objects.